graph_model/trie_model/trie.py

```diff
@@ -29,4 +30,20 @@ def autocomplete(self, prefix, top_n):
         # sort output titles by score
         result.sort(key=lambda item: item['score'], reverse=True)
         print('Normalx search ->\nConsidered: ', len(result), result[:3], '\nPrefix: ',prefix)
-        return result[:top_n]
\ No newline at end of file
+        return result[:top_n]
+
+    def to_dict(self):
+        self.root = self.root.to_dict()
+        print(type(self.root))
+        return self.__dict__
+
+    def load(self, json_dict):
```

Comment:
```suggestion
    def load(self, fileobj:TextIO):
```
Get TextIO from typing module.